### PR TITLE
test fails when run a second time due to persisted data

### LINKED ISSUE #11429: Test Failure: PersistentExecutorWithFailoverEnabledTest.testCreateAndRemoveTaskGroups fails when test run twice
Test Failure: com.ibm.ws.concurrent.persistent.fat.PersistentExecutorWithFailoverEnabledTest.testCreateAndRemoveTaskGroups

```
testCreateAndRemoveTaskGroups:junit.framework.AssertionFailedError: 2020-03-20-17:32:40:141 The response did not contain '[SUCCESS]'.  Full output is:'
SchedulerFATServlet is starting testCreateAndRemoveTaskGroups<br>
<pre>ERROR in testCreateAndRemoveTaskGroups:
java.lang.Exception: Unable to create the third property.
at web.SchedulerFATServlet.testCreateAndRemoveTaskGroups(SchedulerFATServlet.java:1694)
at web.SchedulerFATServlet.doGet(SchedulerFATServlet.java:147)
at javax.servlet.http.HttpServlet.service(HttpServlet.java:687)
at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
at com.ibm.ws.webcontainer.servlet.ServletWrapper.service(ServletWrapper.java:1230)
at com.ibm.ws.webcontainer.servlet.ServletWrapper.handleRequest(ServletWrapper.java:729)
at com.ibm.ws.webcontainer.servlet.ServletWrapper.handleRequest(ServletWrapper.java:426)
at com.ibm.ws.webcontainer.filter.WebAppFilterManager.invokeFilters(WebAppFilterManager.java:1226)
at com.ibm.ws.webcontainer.filter.WebAppFilterManager.invokeFilters(WebAppFilterManager.java:1010)
at com.ibm.ws.webcontainer.servlet.CacheServletWrapper.handleRequest(CacheServletWrapper.java:75)
at com.ibm.ws.webcontainer.WebContainer.handleRequest(WebContainer.java:938)
at com.ibm.ws.webcontainer.osgi.DynamicVirtualHost$2.run(DynamicVirtualHost.java:279)
at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink$TaskWrapper.run(HttpDispatcherLink.java:1134)
at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink.wrapHandlerAndExecute(HttpDispatcherLink.java:415)
at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink.ready(HttpDispatcherLink.java:374)
at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.handleDiscrimination(HttpInboundLink.java:551)
at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.handleNewRequest(HttpInboundLink.java:484)
at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.processRequest(HttpInboundLink.java:346)
at com.ibm.ws.http.channel.internal.inbound.HttpI
```

The attempt to insert a duplicate entry happens because the test case runs twice.  It passes on the first run (which creates the entry for the first time) and fail on the second (entry already exists).  This works fine on Derby because the test case explicitly deletes the Derby database in between runs of the test case.  However, when other databases are used, the data remains in place and the error occurs on the second time through.

### PR DESCRIPTION
**Pull request by njr-11**
pull fixes #11429

### CODE REVIEW

**Comment by njr-11** [comment]
#build

File: dev/com.ibm.ws.concurrent.persistent_fat/fat/src/com/ibm/ws/concurrent/persistent/fat/PersistentExecutorTest.java [hunk 1]
```diff
@@ -38,8 +39,8 @@
     @Server("com.ibm.ws.concurrent.persistent.fat")
     @TestServlet(servlet = SchedulerFATServlet.class, path = APP_NAME)
     public static LibertyServer server;
-    
-    public static final JdbcDatabaseContainer<?> testContainer = FATSuite.testContainer;
+
+    public static final JdbcDatabaseContainer<?> testContainer = DatabaseContainerFactory.create();
```
Comment (KyleAure) [inline dev/com.ibm.ws.concurrent.persistent_fat/fat/src/com/ibm/ws/concurrent/persistent/fat/PersistentExecutorTest.java]: You can add @ClassRule here and remove `testContainer.start()` and stop later on.

File: dev/com.ibm.ws.concurrent.persistent_fat/fat/src/com/ibm/ws/concurrent/persistent/fat/PersistentExecutorWithFailoverEnabledTest.java [hunk 2]
```diff
@@ -43,7 +44,7 @@
     @TestServlet(servlet = SchedulerFATServlet.class, path = APP_NAME)
     public static LibertyServer server;
     
-    public static final JdbcDatabaseContainer<?> testContainer = FATSuite.testContainer;
+    public static final JdbcDatabaseContainer<?> testContainer = DatabaseContainerFactory.create();
```
Comment (KyleAure) [inline dev/com.ibm.ws.concurrent.persistent_fat/fat/src/com/ibm/ws/concurrent/persistent/fat/PersistentExecutorWithFailoverEnabledTest.java]: Same here add `@ClassRule`

**Comment by njr-11** [comment]
#build
#run-libby-bot

**Pull request merged**